File: PERL/Cap03/c03ex03.pl
# c03ex03.pl 

#!/usr/bin/perl

use strict;
use warnings;

print "Entre quantidade de horas trabalhadas ..: ";
my $HT = <STDIN>;
chomp($HT);

print "Entre valor do salario-hora ............: ";
my $VH = <STDIN>;
chomp($VH);

print "Entre valor do percentual de desconto ..: ";
my $PD = <STDIN>;
chomp($PD);

my $SB = $HT * $VH;
my $TD = ($PD / 100) * $SB;
my $SL = $SB - $TD;

printf "\nSalario Bruto ....: %f\n", $SB;
printf "Salario Liquido ..: %f\n\n", $SL;

print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: GO/Cap03/c03ex03.go
// c03ex03.go

package main
import "fmt"

func main() {
  var HT, VH, PD, SB, TD, SL float64

  fmt.Print("Entre quantidade de horas trabalhadas ..: ")
  fmt.Scan(&HT)

  fmt.Print("Entre valor do salario-hora ............: ")
  fmt.Scan(&VH)

  fmt.Print("Entre valor do percentual de desconto ..: ")
  fmt.Scan(&PD)

  SB = HT * VH
  TD = (PD / 100) * SB
  SL = SB - TD

  fmt.Println()

  fmt.Printf("Salario Bruto ....: %.2f\n", SB)
  fmt.Printf("Salario Liquido ..: %.2f\n", SL)

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}
